pkg/systemdgen/systemdgen.go

```diff
@@ -0,0 +1,40 @@
+package systemdgen
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"path/filepath"
+)
+
+var template string = `[Unit]
+Description=%s Podman Container
+[Service]
+Restart=%s
+ExecStart=/usr/bin/podman start %s
+ExecStop=/usr/bin/podman stop %s
+KillMode=none
+Type=forking
+PIDFile=%s
```

Comment:
I've tested this config and I confirm it is what works the best for us in OpenStack.